File: src/ViewModels/UserManager/UserVM.cs
namespace ViewModels.UserManager;

public class UserVM
{
    public string Id { get; set; } = string.Empty;
    public string UserName { get; set; } = string.Empty;
    public string Email { get; set; } = string.Empty;
    public string PhoneNumber { get; set; } = string.Empty;
    public string FullName { get; set; } = string.Empty;
    public DateTime DateOfBirth { get; set; }
    public DateTime CreatedDate { get; set; }
    public DateTime? UpdatedDate { get; set; }
}

File: src/SPA/src/app/_core/models/user-manager/uservm.ts
export interface UserVM {
  id: string;
  userName: string;
  email: string;
  phoneNumber: string;
  fullName: string;
  gender: number; // 0 cho Male (nam), 1 cho Female (nu)
  isActive: boolean;
  lastLoginTime: Date | string | null;
  dateOfBirth: Date | string;
  createdDate: Date | string;
}

